docs/appendix/zwe_server_command_reference/zwe/zwe-install.md

```diff
@@ -6,13 +6,10 @@
 
 ## Description
 
-After you extract Zowe convenience build, you can run this command to install
+After installation via SMP/E  Zowe convenience build, run this command to install
```

Comment:
Modify this, to: After installation of Zowe via SMP/E, PSWI or convenience build, run this command to install sample modules and some load modules into MVS data sets.